index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lembani Sakala - Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+        integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <header class="header">
+        <nav class="nav">
+            <div class="brand"><a href="#">Welcome</a></div>
+            <div class="menu"><i class="fa fa-bars fa-xl"></i></div>
+        </nav>
+    </header>
+
+    <div class="headline">
+        <div class="headline-content">
+            <span class="intro-1">Hey there. I'm Lembani</span>
+            <br>
+            <span class="intro-2">I'm a software developer</span>
+        </div>
+        <div class="description">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don't hestiate to contact me.
+        </div>
+        <div class="social-icons">
+            <i class="social-icon fa-brands fa-twitter fa-xl"></i>
+            <i class="social-icon fa-brands fa-linkedin fa-xl"></i>
+            <i class="social-icon fa-brands fa-medium fa-xl"></i>
+            <i class="social-icon fa-brands fa-github fa-xl"></i>
+            <i class="social-icon fa-brands fa-angellist fa-xl"></i>
+        </div>
```

Comment:
- [ ] Kindly wrap the social icons in `ul tag` to group them and` li per icon` with `a tag`.